hedyorg/hedy
# 💻 customize class page enhancement

Fixes #3976 
**How to test?**
- go to the customize-class page
- see the new layout, and try it out!
